### PR TITLE
fix: comment syntax in the core config file

### PR DESCRIPTION
**Pull request by antouhou**
## Issue being fixed or feature implemented

Incorrect core config file syntax that resulted in core not starting RPC server

## What was done?

Changed comment syntax in the config files

## How Has This Been Tested?

Run the bootstrap

## Breaking Changes

None

## Checklist:

- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have added or updated relevant unit/integration/functional/e2e tests
- [x] I have made corresponding changes to the documentation

**For repository code-owners and collaborators only**
- [x] I have assigned this pull request to a milestone

### CODE REVIEW

File: configs/evonet/core/dashd.conf [hunk 1]
```diff
@@ -37,7 +37,7 @@ allowprivatenet=0
 
 printtoconsole=1
 
-[devnet]
```
Comment (thephez) [inline configs/evonet/core/dashd.conf:40]: This should work as of 0.16 (https://dashcore.readme.io/docs/dash-core-wallet-configuration-file). It was backported from [Bitcoin](https://bitcoincore.org/en/releases/0.17.0/#configuration-sections-for-testnet-and-regtest)

Comment (antouhou) [reply configs/evonet/core/dashd.conf:40]: Well, this branch still uses 0.15

Comment (antouhou) [reply configs/evonet/core/dashd.conf:40]: We need to update it than

**Comment by antouhou** [comment]
Close this PR as it was fixed in #155

**Pull request closed without merging**